WebRoot/js/zimbraAdmin/accounts/model/ZaAccount.js

```diff
@@ -1721,6 +1731,23 @@ function(by, val) {
     } else {
         this[ZaAccount.A2_autodisplayname] = "FALSE";
     }
+    let arr=["domainAdminZimletRights","adminLoginCalendarResourceAs","domainAdminConsoleRights"];
+    var count=0;
+    if( allGrantsList.length >= 3){
+    for(var i=0; i< allGrantsList.length; i++){
+        if(allGrantsList[i].grantee_id === this.id){
+            if((arr.includes(allGrantsList[i].right)) && (allGrantsList[i].deny === "0")){
+                count++;
+            }
+        }
+        
+    }
+}else{
+    count=0;
+}
```

Comment:
indent

Reply:
moved these changes to https://github.com/Zimbra/zm-delegated-admin-zimlet/pull/8